assets/javascripts/modules/checkout/submit.js

```diff
@@ -25,16 +27,20 @@ define([
              paymentErrorMessages,
              stripeCheckout,
              raven,
-             formElements) {
+             formElements,
+             ophan,
+             cookie) {
     'use strict';
 
     function submitHandler() {
         if (formEls.$CHECKOUT_SUBMIT.length) {
-            formEls.$CHECKOUT_SUBMIT[0].addEventListener('click', submit, false);
+            formEls.$CHECKOUT_SUBMIT[0].addEventListener('click', function(e) {
+                ophan.loaded.then(function(o) { submit(e, o.viewId) });
```

Comment:
Thinking about this line, I'm wondering what happens if Ophan doesn't load - will that prevent submission of the form?

Reply:
OK I've addressed this now!